wis2box-create-config.py

```diff
@@ -284,6 +284,39 @@ def get_wis2box_url() -> str:
     return wis2box_url
 
 
+def get_custom_ui_logo():
+    """
+    Prompt the user to enter a custom UI logo path or URL.
+
+    :returns: string of URL logo
+    """
+    while True:
+        logo = input("Enter the URL for the custom UI logo (leave blank to skip): ").strip() # noqa
+        msg = f"Confirm custom UI logo: '{logo or 'Default'}'? (y/n): "
+        confirm = input(msg).strip().lower()
+        if confirm == 'y':
+            return logo if logo else None
+
+
+def get_default_ui_language():
```

Comment:
```python
def get_default_ui_language() -> Union[str, None]:
```

Note that you will need to update `from typing import Tuple` to `from typing import Tuple, Union` on line 29.